estate/__manifest__.py

```diff
@@ -0,0 +1,6 @@
+{'name': 'Real Estate',
+'depends': ['base'],
+'application': True,
+'author': 'leker',
+'license': 'LGPL-3',
+}
```

Comment:
let's go with this format 👀 
```suggestion
{
    'name': 'Real Estate',
    'depends': ['base'],
    'application': True,
    'author': 'leker',
    'license': 'LGPL-3',
}
```

Reply:
thanks! done, added in new commit